client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -49,14 +50,20 @@ class MobileLayout extends Component<MobileLayoutProps> {
       hasPreview,
       preview,
       guideUrl,
-      videoUrl
+      videoUrl,
+      usesMultifileEditor
     } = this.props;
 
     const editorTabPaneProps = {
       mountOnEnter: true,
       unmountOnExit: true
     };
 
+    // This is not the same logic as the desktop version since the mobile
+    // version does not have an ActionRow, but does need a way to switch between
+    // the different tabs.
+    const displayEditorTabs = showUpcomingChanges && usesMultifileEditor;
```

Comment:
Why not just `usesMultiFileEditor` here?

Reply:
We could, but this is a (minor) way to make sure we don't accidentally show the tabs until we're ready.